avm/res/cache/redis-enterprise/tests/e2e/active-geo-replication/main.test.bicep

```diff
@@ -0,0 +1,100 @@
+targetScope = 'subscription'
+
+metadata name = 'Active geo-replication'
+metadata description = 'This instance deploys the module with active geo-replication enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-cache-redisenterprise-${serviceShort}-rg'
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'creagr'
+
+@description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. The name of the geo-replication group.')
+param geoReplicationGroupName string = 'geo-replication-group'
+
+@description('Optional. The zones to deploy resources to.')
+param zones array = [
+  1
+  2
+  3
+]
+
+// ============ //
+// Variables    //
+// ============ //
+var redisName1 = '${namePrefix}${serviceShort}001'
+var redisName2 = '${namePrefix}${serviceShort}002'
```

Comment:
Would be great to move them to the place they're used first (which should be the dependency deployments) and return the values from there to reference in the test. I'm aware this is a lot more complicated than simply having them as variables/parameters, but consistency-wise it's quite helpful to follow a consistent pattern of where values are defined etc. Happy to discuss though if you really don't like that 😄

Reply:
The `zones` specification can probably be removed in its entirety as you defined it as a default value in the dependency deployment which should do the trick.

Reply:
Sure, can you take another look?

Reply:
Just did and added one last comment. Great work @JeffreyCA, will resolve.